### ISSUE TITLE
Conflicting peer dependency: @glimmer/tracking@2.0.0-beta.21

### ISSUE THREAD

**Issue by prestoncobb** [josemarluedke/glimmer-apollo#71]
In a project I am working on, we are currently running into an issue with conflicting peer dependancies from glimmer-apollo, which causes installs to fail. Glimmer-apollo appears to require `@glimmer/tracking@2.0.0-beta.21`, but our application (and other addons) are on `v1.1.2`. Is there a resolution any can recommend? I'd like to resolve this properly instead of relying on `--force` or `--legacy-peer-deps`

Ember Version: 4.4
Node Version: 14.20.1
NPM Version: 8.19.3
glimmer-apollo Version: 0.5.1 

<details>
<summary>Full Error:</summary>

```node
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: glimmer-apollo@0.5.1
npm ERR! Found: @glimmer/tracking@1.1.2
npm ERR! node_modules/@glimmer/tracking
npm ERR!   dev @glimmer/tracking@"^1.0.4" from the root project
npm ERR!   @glimmer/tracking@"^1.0.2" from ember-apollo-client@3.2.1
npm ERR!   node_modules/ember-apollo-client
npm ERR!     dev ember-apollo-client@"^3.2.0" from the root project
npm ERR!   4 more (ember-basic-dropdown, ember-concurrency, ...)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @glimmer/tracking@"^2.0.0-beta.15" from glimmer-apollo@0.5.1
npm ERR! node_modules/glimmer-apollo
npm ERR!   dev glimmer-apollo@"^0.5.1" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: @glimmer/tracking@2.0.0-beta.21
npm ERR! node_modules/@glimmer/tracking
npm ERR!   peer @glimmer/tracking@"^2.0.0-beta.15" from glimmer-apollo@0.5.1
npm ERR!   node_modules/glimmer-apollo
npm ERR!     dev glimmer-apollo@"^0.5.1" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.

```
</details>

**Comment by bertdeblock** [comment]
Maybe `@glimmer/tracking@2.0.0-beta.21` is needed for Glimmer apps, but we should allow v1 for Ember apps?

**Comment by johanrd** [comment]
this should now be fixed after https://github.com/josemarluedke/glimmer-apollo/pull/96

@prestoncobb please check [v0.6.6](https://github.com/josemarluedke/glimmer-apollo/releases/tag/v0.6.6) to see if you agree.

**Comment by prestoncobb** [comment]
> this should now be fixed after #96
> 
> @prestoncobb please check [v0.6.6](https://github.com/josemarluedke/glimmer-apollo/releases/tag/v0.6.6) to see if you agree.

Everything looks to be working without any overrides. Thank you!

**Comment by prestoncobb** [comment]
Resolved in [v0.6.6](https://github.com/josemarluedke/glimmer-apollo/releases/tag/v0.6.6)
